s3-discovery/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-bullseye
+
+WORKDIR /app
+
+RUN pip install --upgrade pip
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+RUN find . -type f -name '*.pyc' | while read f; do n=$(echo $f | sed 's/__pycache__\///' | sed 's/.cpython-[2-3][0-9]//'); cp $f $n; done;
+RUN find . -type d -a -name '__pycache__' -print0 | xargs -0 rm -rf
+RUN find . -type f -a -name '*.py' -print0 | xargs -0 rm -f
+RUN find . -type d -a -name 'tests' -print0 | xargs -0 rm -rf
+RUN echo "Remove lambda python packages"
+RUN rm -rdf ./numpy/doc/
+RUN rm -rdf ./stack
+RUN rm -rdf ./docutils*
+
+COPY handler.py handler.py
```

Comment:
Just an FYI that I created an issue to explore if we can de-dupe some of these Dockerfiles, https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/issues/69 feel welcome to add any ideas or thoughts you have there about de-duplication